Automattic/wp-calypso
# Build: Move Babel plugin deps to calypso-build package

#### Changes proposed in this Pull Request

I told @jeffersonrabb that it should be possible to remove babel plugin deps from Newspack's `package.json` since `calypso-build` was pulling them in anyway.

Turns out I lied :sweat_smile:  -- `calypso-build` hasn't been listing all its dependent Babel plugins that its `babel.config.js` makes use of.

This PR is supposed to fix that:

- Plugins that are only used by `calypso-build` are moved there from the root `package.json`
- Plugins that are still _also_ used by the top-level `babel.config.js` are duplicated in both `package.json`'s

#### Testing instructions

```
npm run distclean
npm ci
npm start
```

Does Calypso successfully build and run?

Verify that `calypso-build`'s `package.json` now contains all the Babel plugins and presets mentioned in its `babel.config.js`.
